priv/templates/measurements.html.eex

```diff
@@ -0,0 +1,179 @@
+<h2>Results</h2>
+<h3>Basic metrics</h3>
+<div id="basicMetrics"></div>
+<h3>Utilization by scheduler type</h3>
+<div id="schedulersByType"></div>
+<h3>Total scheduler utilization</h3>
+<div id="totalSchedulers"></div>
+<h3>Utilization by scheduler type and ID</h3>
+<div id="normalScheduler"></div>
+<div id="cpuScheduler"></div>
+<div id="ioScheduler"></div>
+<script>
+  <%= dont_compare = is_nil(base_measurements) %>
+  let indicatorConfig =  {
+    type: "indicator",
+    number: { font: { size: 70 }},
+    <%= if dont_compare do %>
+    mode: "number"
+    <% else %>
+    mode: "number+delta"
+    <% end %>
+  };
+  let deltaConfig = {
+    relative: true,
+    valueformat: ".2%",
+    increasing: { color: "#FF4136" },
+    decreasing: { color: "#3D9970" }
+  };
+  let layoutConfig = {
+    margin: {t: 0, b: 0, l: 0, r: 0},
+    height: 200
+  };
+  let plotlyConfig = {
+    displaylogo: false,
+    responsive: true
+  };
+
+  let basicMetrics = [
+    {
+      title: { text: "Reductions" },
+      value: <%= new_measurements.reductions %>,
+      <%= unless dont_compare do %>
+      delta: { reference: <%= base_measurements.reductions %>, ...deltaConfig },
+      <% end %>
+      domain: { row: 0, column: 0 },
+      ...indicatorConfig
+    },
+    {
+      title: { text: "Context switches" },
+      value: <%= new_measurements.context_switches %>,
+      <%= unless dont_compare do %>
+      delta: { reference: <%= base_measurements.context_switches %>, ...deltaConfig },
+      <% end %>
+      domain: { row: 0, column: 1 },
+      ...indicatorConfig
+    }
+  ];
+
+  let basicMetricsLayout = {
+    grid: { rows: 1, columns: 2, pattern: "independent" },
+    ...layoutConfig
+  };
+
+  Plotly.newPlot("basicMetrics", basicMetrics, basicMetricsLayout, plotlyConfig);
+
+  let schedulersByType = [
+    <%= for {metric, title, column} <- [
+      {:total_normal, "Normal Scheduler", 0},
+      {:total_cpu, "CPU Scheduler", 1},
+      {:total_io, "IO Scheduler", 2}
+    ] do %>
+      {
+        title: { text: "<%= title %> [s]" },
+        value: <%= elem(new_measurements.scheduler_info[metric], 0) %>,
+        <%= unless dont_compare do %>
+        delta: { reference: <%= elem(base_measurements.scheduler_info[metric], 0) %>, ...deltaConfig },
+        <% end %>
+        domain: { row: 0, column: <%= column %> },
+        ...indicatorConfig
+      },
+    <% end %>
+  ];
+
+  let schedulersByTypeLayout = {
+    grid: { rows: 1, columns: 3, pattern: "independent" },
+    ...layoutConfig
+  };
+
+  Plotly.newPlot("schedulersByType", schedulersByType, schedulersByTypeLayout, plotlyConfig);
+
+  let totalSchedulers = [
+    {
+      title: { text: "Total utilization [s]" },
```

Comment:
I think that we should present this in % at first. Seconds can be in `()` so e.g. 

```
Total utilization
50% (5s)
```

Please check other places too.

Reply:
How about making separate gauges for utilization in percents and secods? The problem with displaying both at once is that the plotting library we use doesn't support multiple values if we want them to be compared and formatted.

Reply:
I see, so let's try to create separate gauges.